src/Service/EloquaApiClient.php

```diff
@@ -367,7 +371,7 @@ private function getEloquaApiCache($type) {
   public function doEloquaApiRequest($verb, $endpoint, $body = NULL, $queryParams = NULL) {
     // TODO Ideally merge all the Guzzle requests into one generic method.
     // Guzzle Client.
-    $guzzleClient = new GuzzleClient([
+    $guzzleClient = $this->httpClientFactory->fromOptions([
```

Comment:
surely this shouldnt be `$guzzleClient` as per the line 46 where its `protected $httpClientFactory;`